engineyard/ey-cookbooks-stable-v5
# [FB-856] Update Percona-Server Version

Description of your patch
-------------
Updates the default mysql/percona-server version of 5.6 and 5.7 to `5.6.45.86.1` and `5.7.27.30` respectively. 

Recommended Release Notes
-------------
Updates the default percona-server version of MySQL versions 5.6 and 5.7 to `5.6.45.86.1` and `5.7.27.30` respectively. 

Estimated risk
-------------

Components involved
-------------
`mysql` cookbook

Description of testing done
-------------
1. Boot a new environment with the labrea stack release, built from the changes made for the `mysql` cookbook
2. Check for the latest available mysql version available for MySQL version 5.6 and 5.7
3. Verify if the MySQL version matches to `5.6.45.86.1` in 5.6 and `5.7.27.30` in 5.7

QA Instructions
-------------
1. Check the default versions by booting a solo environment for MySQL v5.5, v5.6 and v5.7
2. Check the default versions for a multi-environment for MySQL versions v5.5, v5.6 and v5.7
